packages/gitbook/src/lib/tracking.ts

```diff
@@ -43,6 +54,10 @@ export async function serveProxyAnalyticsEvent(req: Request) {
         method: 'POST',
         headers: {
             'Content-Type': 'application/json',
+            'x-location-country': country || '',
```

Comment:
```suggestion
            ...(country ? { 'x-location-country': country } : {}),
```

would omitting the header be better than the empty string? Fine either way